Dockerfile.assisted-installer

```diff
@@ -1,8 +1,16 @@
-FROM quay.io/centos/centos:centos8
-copy build/installer /
+FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.7 AS builder
+ENV GOFLAGS=-mod=mod
+WORKDIR /go/src/github.com/openshift/assisted-installer
+COPY . .
```

Comment:
Maybe add only the required files (`src`, `go.mod`, `go.sum`) to reduce the image size and make caching work better. Alternatively, add a `.dockerignore` file